### PR TITLE
Add pop-out explanation for Python layers

### LINKED ISSUE #636: Add better Python layer documentation
The documentation from https://github.com/NVIDIA/DIGITS/pull/447 is great and all, but we should have better documentation readily available within the app, like https://github.com/NVIDIA/DIGITS/pull/628#issuecomment-195137527.

### PR DESCRIPTION
**Pull request by lukeyeager**
*Close #636*

Cleanup Python layer upload UI and add some pop-out text similar to https://github.com/NVIDIA/DIGITS/pull/628#issuecomment-195137527.

Also add network outputs with names containing "accuracy" to the accuracy axis in graphs. It's a little hacky, but I don't think we need to find a general purpose solution to that problem just yet.

### CODE REVIEW

File: digits/templates/models/python_layer_explanation.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
+{# Copyright (c) 2015-2016, NVIDIA CORPORATION.  All rights reserved. #}
+
+<h1>Using Python layers</h1>
+
+<p>
+    Caffe lets you can define custom layers with Python.
```
Comment (gheinrich) [inline digits/templates/models/python_layer_explanation.html]: small typo here

**Comment by gheinrich** [comment]
This is working for me (I tried both the "blank square" and "accuracy" layers examples on MNIST+LeNet).

File: digits/templates/models/python_layer_explanation.html (hunk 1 above)
Comment (lukeyeager) [inline digits/templates/models/python_layer_explanation.html]: Haha whoops - right at the beginning! Thanks.

**Comment by lukeyeager** [comment]
Thanks for the review, Greg. Here's a screenshot of the feature for those who are interested:

python-layer-explanation

**Pull request merged**